models/ArticleTag.js

```diff
@@ -0,0 +1,14 @@
+export default (sequelize, DataTypes) => {
+  const ArticleTag = sequelize.define('ArticleTag', {
+    articleId: {
+      type: DataTypes.INTEGER,
+      primaryKey: true
+    },
+    tagName: {
```

Comment:
#### Nitpick
- Why do you have `tagName` as primary key

Reply:
The reason for that is because the default unused primaryKey `id` was removed from the joined table.

Reply:
@devdbrandy Why is it removed?

Reply:
On the association, we are making use of the attribute `tagName` as the primary key, so the 'id` field is not relevant. However, I've included that with my most recent commit update.

Reply:
@devdbrandy , I thought `Sequelize` creates `articleId` automatically when you create the association?

Reply:
Wow @aanchirinah you are very correct. I have removed both `articleId`,  and `tagName` attributes from the `ArticleTag` model. Thanks for the helpful tip!